tests/ViewComposers/TestCountryLocaleViewComposer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace PodPoint\Countries\Tests\ViewComposers;
+
+use PodPoint\Countries\ViewComposers\CountryLocaleViewComposer;
+use Illuminate\View\View;
+use Illuminate\Config\Repository;
+use PHPUnit\Framework\TestCase as PHPUnitTestCase;
+
+class TestCountryLocaleViewComposer extends PHPUnitTestCase
+{
+    /**
+     * Makes sure every methods of the view composer are called properly.
+     */
+    public function testCompose()
+    {
+        /** @var Repository|\PHPUnit_Framework_MockObject_MockObject $configMock */
+        $configMock = $this->getMockBuilder(Repository::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['get'])
+            ->getMock();
+
+        $countryLocaleViewComposer = new CountryLocaleViewComposer($configMock);
+
+        /** @var View|\PHPUnit_Framework_MockObject_MockObject $viewMock */
+        $viewMock = $this->getMockBuilder(View::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['with'])
+            ->getMock();
+
+        $viewMock->expects($this->once())
+            ->method('with');
```

Comment:
It feels like the only thing worth testing in this class is that the `countryLocalesOptions` value passed to the `with` method is in the right array format but its not being tested? TBH I wouldn't even bother writing a test for this composer.

Reply:
Yeah I just test that all methods are called properly. I thought that testing that the array was well formed was too much because it's just getting informations from a config file and put them into an array.